### DISCUSSION TITLE
Why not output tailwind classnames?

### DISCUSSION THREAD

**Discussion by wmertens** [chakra-ui/panda#1181] [General]
Since both panda and tailwind are atomic CSS, would it not be better if the default output of Panda uses TW-compatible classnames?

That way, library builders could export components that are usable by Panda users, but also by TailWind, Uno etc. users.

**Comment by wmertens** [comment]
...or maybe at least output tw-compatible classes where there's a 1-1 correspondence?

At the very least, in a mixed project, it would mean that several classes would be shared, thereby reducing the total CSS.

**Reply by astahmer** [reply]
you could definitely make a preset for that, I'm sure eventually it'll be done

**Reply by wmertens** [reply]
So how would that work? Can you influence the class name choice in a preset, or would it be post-processing?

**Reply by astahmer** [reply]
yes you can influence the className from a preset, by extending/overriding the utilities 
https://panda-css.com/docs/customization/utilities

this would require overriding the className for each utilities so I'm not saying this is a quick job to do, but still it can be done

**Reply by wmertens** [reply]
So hence my question, since the Panda classnames are arbitrary anyway, why not make them the same as TW classnames whenever they're the same? :-)
